components/KeyboardScrollView.js

```diff
@@ -29,10 +26,10 @@ export const KeyboardScrollView = (props) => {
     <KeyboardAvoidingView
       behavior={behavior}
       keyboardVerticalOffset={defaultVerticalOffset}
-      style={stylesOverride}
```

Comment:
As we always need this to be the same as it ensure button is at bottom of screen I just removed it so we aren't repeating lots of code